Functions/Import/xASL_imp_DCM2NII_ConvertScan.m

```diff
@@ -77,16 +79,21 @@
         fprintf('Subject = %s, visit = %s, session = %s, scan = %s\n',scanFields.subjectID, scanFields.visitID, printSession, scan_name);
     end
 
-    % Defaults
-    bOneScanIsEnough = false;
-    bPutInSessionFolder = true; 
+%     % Defaults
+    bOneScanIsEnough = false; % PM: THIS USED TO BE TRUE FOR ANATOMICAL SCANS
+    bPutInSessionFolder = true; % THIS USED TO BE FALSE FOR ANATOMICAL SCANS
     switch scan_name
         case {'ASL4D', 'M0', 'ASL4D_RevPE', 'func_bold'}
             bPutInSessionFolder = true;
-        case {'T1', 'WMH_SEGM', 'FLAIR', 'T2', 'T1c'}
-            bPutInSessionFolder = false;
+            bAnatomical = false;        
+        case {'T1w' 'T2w' 'T1', 'WMH_SEGM', 'FLAIR', 'T2', 'T1c'}
```

Comment:
isn't scan_name a keyword? So if you change it here, you might also need to make sure that this is correctly recognized in NII2BIDS. Is there a support for both T1w and T1?

Reply:
Yes we probably need to do this. The problem is that it is confusing if we should use T1 or T1w in import, whereas T1w is the BIDS term (and we use both throughout). `scan_name` is not a BIDS keyword afaik

Reply:
Actually, after some though. Yes - OK to use this as this will be renamed to BIDS that has a well defined name for it. 
And I have seen that you have made some further modifications later. 

However, two things have to be done:
1. We need to check how this behaves for T1c. Because previously we had T1w and T1c - which are quite distinct. Now with T1 and T1w, there might be issues that T1c is recognized incorrectly as T1.
2. Functions/BIDS/xASL_bids_Config.m - line 187 and 193 have to be adapted by adding the T1 and T2 types.

Reply:
In general: we didn't test it anyway for `T1c`, but we can of course adapt regular expressions to exclude `T1c` when we mean `T1|T1w`.

Here, there is no different behavior for T1|T1w|T1c, this line only says that all structural stuff are `bAnatomical`.

`xASL_bids_Config` line 187 is correct (it has `T1w|T2w|T1c`)

Line 192: can it be `T1(|w)` under filename? This line was already incompatible with line 187, by not having `T1c`?

BTW: my change in `xASL_imp_DCM2NII_ConvertScan` just made sure that both `T1w` and `T1` will be interpreted as anatomical; doesn't necessarily mean that the whole import needs to be able to recognize T1 and T1w (although that would be nice).

Can you fix this further?